app/views/space_invitation_mailer/space_invitation_email.text.erb

```diff
@@ -6,4 +6,4 @@ You may view, accept or ignore this invitation at <%= polymorphic_url(@invitatio
 
 We hope to see you there! 🥳🎈🎉
 
-- The <%= @invitation.space.name %> Crew
+- Convene Support on behalf of <%= @invitation.invitor_display_name %>
```

Comment:
I would prefer if we kept it as `The {space_name} Crew on behalf of {invitor_display_name}` since it's really about inviting them to a specific space.

Reply:
fixed in 7eea88dc691af50a7ec30540f1fb8cee0066103b